lib/assertions/directory_content.js

```diff
@@ -0,0 +1,213 @@
+module.exports = function (chai, utils) {
+
+	var Assertion = chai.Assertion;
+	var AssertionError = chai.AssertionError;
+	var flag = utils.flag;
+	var assert = chai.assert;
+
+	var readdir = require('readdir-enhanced');
+	var format = require('util').format;
```

Comment:
@keithamus - Yeah, I used Chai's built-in message formatting (i.e. `#{this}`, `#{exp}`, etc.) wherever possible, but in some cases I needed to inject variables other than the ones that Chai has placeholders for, which is why I fell-back to `util.format()` in those cases.

Or am I mistaken, and Chai supports arbitrary placeholders for arbitrary variables?